.github/workflows/system.yml

```
@@ -0,0 +1,28 @@
+---
+name: System tests
```

This ends up showing in the UI of all PRs, and currently it's Leading Caps in all workflows in all repos I think. Would be good to be consistent.

```suggestion
name: System Tests
```

Also, I wonder if anyone has a suggestion for a more descriptive workflow/job name than "System Tests/Test"? I get the idea, but it took some thinking and I authored a lot of these tests. :stuck_out_tongue_winking_eye: A more verbose job name maybe? /me is struggling to think of a good suggestion, will follow up if I think of one

---

Updated the caps, I'm not married to the name if you have a better suggestion I don't mind changing it

---

Unfortunately only the job name is displayed in the branch settings tab so having that more descriptive helps. I suggest job name "_Deployment Test_", so full  name "_System Tests/Deployment Test_".